README.md

```diff
@@ -40,10 +41,16 @@ brief description of what it does.
 * [cubeseer](https://github.com/justinj/cubeseer) - Diagram generator, but contains a cube representation
 * [RaMSDeL](https://github.com/justinj/ramsdel) - Random Move Scrambler Definition Language
 
-#C++
-
+C++
+===
```

Comment:
The C# one needed some weird escaping for the # to show up using the other way, so it's more readable like this. I guess the right thing would be to change all the titles to use this format.